packages/BetterPhpDocParser/PhpDoc/ArrayItemNode.php

```diff
@@ -22,7 +22,7 @@ public function __toString(): string
     {
         $value = '';
 
-        if ($this->key !== null) {
+        if ($this->key !== null && ! is_numeric($this->key)) {
```

Comment:
this is required as the `__toString()` cause invalid change `0=@OA` conversion on generic:

```php
     * @OA\Property(
     *     type="array",
     *     @OA\Items(ref=@Model(type=TestItem::class))
     * )
```

Reply:
updated with other usage with `is_int($key)` to make consistent at PR:

- https://github.com/rectorphp/rector-src/pull/5284